pkgs/development/libraries/qt-6/modules/qtwayland.nix

```diff
@@ -36,4 +37,8 @@ qtModule {
       sha256 = "sha256-1EIcMj6+yIpqXAGZB3ZbrwRkl4n1o7TVP2SC1Nu1t78=";
     })
   ];
+
+  meta = {
+    platforms = lib.subtractLists lib.platforms.darwin lib.platforms.unix;
```

Comment:
Small nit, and maybe inaccurate, but this might be a good case for `badPlatforms`?

Reply:
I'm not even sure darwin should be excluded, since https://github.com/owl-compositor/owl exists. But sure

Reply:
> Small nit, and maybe inaccurate, but this might be a good case for `badPlatforms`?

I don't understand why qtwayland is not available on Darwin in the first place, and what's wrong with adding it unconditionally...

Reply:
> I don't understand why qtwayland is not available on Darwin in the first place

You are aware that Darwin is not a freedesktop platform and therefore has next to no support for wayland, right?

Reply:
> > I don't understand why qtwayland is not available on Darwin in the first place
> 
> You are aware that Darwin is not a freedesktop platform and therefore has next to no support for wayland, right?

Yes, but from some reason I always thought that a packages that add `qtwayland` to their `buildInputs` still build on Darwin...